doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -183,6 +183,8 @@ The following table indicates the software maturity levels of the support for ea
      - nRF52840
      - nRF5340
      - nRF54H20
+     - nRF54L05
```

Comment:
General question: if we add a HW here, and mark a protocol as not supported, do we still need to add that HW to that protocol specific table and mark all protocol features as not supported? IMO that does not make sense, but I see we do it for some HW. Looks inconsistent now, we should clean it up, may be in separate PR.